common/server-sent-events.conf

```diff
@@ -1,8 +1,3 @@
-proxy_buffering off;
-proxy_cache off;
```

Comment:
So we are now removing:
```
-proxy_buffering off;
-proxy_cache off;
```

I don't remember this being part of the discussion on slack. Is that what the newly linked URL talks about? I'm pretty sure you had to add these to get things working for the debug APIs.